_plugins/jekyll_get.rb

```diff
@@ -25,7 +25,8 @@ def generate(site)
             site.data[d['data']] = source
           end
           if d['cache']
-            path = "#{site.config['data_source']}/#{d['data']}.json"
+            data_source = (site.config['data_source'] or '_data')
```

Comment:
Mind using `||` instead of `or`? Same below.

https://github.com/bbatsov/ruby-style-guide#no-and-or-or